src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -30,13 +30,28 @@ class ReviewCollapsibleChapter extends React.Component {
   constructor(props) {
     super(props);
     this.handleEdit = this.handleEdit.bind(this);
+    this.handleChapterClick = this.handleChapterClick.bind(this);
+
+    this.state = {
+      chapterOpen: false,
+    };
```

Comment:
This component needs to keep track of its own open state now that it is handling the scroll behavior.